src/utils/keyManagement.js

```diff
@@ -2,7 +2,7 @@
 
 // Parse window.location to retrieve holoPort's HC public key (3rd level subdomain in URL)
 const getHcPubkey = () => {
-  return ((process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')
+  return ((process.env.USE_REAL_PUB_KEY === 'true')
```

Comment:
I think this won't work because VUE requires env variables to be prefixed with `VUE_APP_`

```suggestion
  return ((process.env.VUE_APP_USE_REAL_PUB_KEY === 'true')
```

Reply:
Maybe I shouldn't multitask during standup :(